genesis/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tempfile = { workspace = true }
 
 [dev-dependencies]
 solana-runtime = { workspace = true, features = ["dev-context-only-utils"] }
+solana-stake-interface = { workspace = true, features = ["borsh"] }
```

Comment:
Sorry, it's not clear from this PR or the code in `genesis` -- why is this dependency needed?